tests/Fixtures/resources.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<resources>
+    <configdummy>
```

Comment:
Might be better and feels more close to the symfony service definition. In this case what about setting the class as an attribute?

``` 
        <resource id="configDummy" class="Entity\ConfigDummy">
        </resource>
```